src/Command/PelagosClearErrorAndFileCommand.php

```diff
@@ -22,18 +22,30 @@ class PelagosClearErrorAndFileCommand extends Command
      */
     protected $entityManager;
 
+    /**
+     * Used to inject homedirPrefix from .env parameters.
+     *
+     * @var string $homedirPrefix
+     */
+    protected $homedirPrefix;
+
+    /**
+     * Used to inject datastore directory from .env parameters.
+     *
+     * @var string $dataStoreDirectory
+     */
+    protected $dataStoreDirectory;
+
     /**
      * Class constructor for dependency injection.
      *
      * @param EntityManagerInterface $entityManager A Doctrine EntityManager.
      */
-    public function __construct(
-        EntityManagerInterface $entityManager
-    ) {
+    public function __construct(EntityManagerInterface $entityManager, string $homedirPrefix, string $dataStoreDirectory)
```

Comment:
Inject datastore here instead of the paths.

Reply:
still need homedirprefix, but removed datastore path. I still might move that section out to a util.

Reply:
A bit more expensive, but you can also use 'has' from data store.